app/components/global_threads/thread_footer/thread_footer.tsx

```diff
@@ -132,18 +132,21 @@ function ThreadFooter({actions, currentUserId, intl, location, testID, theme, th
 
     // threadstarter should be the first one in the avatars list
     const participants = React.useMemo(() => {
-        let isThreadStarterFound = false;
-        const participantIds = thread.participants.flatMap((participant) => {
-            if (participant.id === threadStarter?.id) {
-                isThreadStarterFound = true;
-                return [];
+        if (thread.participants?.length) {
+            let isThreadStarterFound = false;
+            const participantIds = thread.participants.flatMap((participant) => {
+                if (participant.id === threadStarter?.id) {
+                    isThreadStarterFound = true;
+                    return [];
+                }
+                return participant.id;
+            });
```

Comment:
oh cool! didn't know about that trick, using `flatMap` to `map` and `filter` at the same time!